tests/pkg/utils/mco_deploy.go

```diff
@@ -349,6 +353,11 @@ func CheckAllPodsAffinity(opt TestOptions) error {
 	}
 
 	for _, pod := range podList {
+		if pod.Labels["name"] == "endpoint-observability-operator" || pod.Labels["component"] == "metrics-collector" ||
+			pod.Labels["component"] == "uwl-metrics-collector" {
+			// No affinity set for endpoint-operator and metrics-collector in the hub
+			continue
+		}
```

Comment:
Why these changes? Why do we skip looking at affinities of these pods? They still exist and can have affinities, from what I know.

Reply:
We should eventually have test coverage for this.